pyproject.toml

```diff
@@ -1,10 +1,3 @@
-[tool.setuptools]
```

Comment:
If you remove these files
`pip-compile -v -U --resolver=backtracking pyproject.toml`
won't work anymore.